src/ray/common/ray_config_def.h

```diff
@@ -193,6 +193,11 @@ RAY_CONFIG(int, object_manager_repeated_push_delay_ms, 60000)
 /// chunks exceeds the number of available sending threads.
 RAY_CONFIG(uint64_t, object_manager_default_chunk_size, 1000000)
 
+/// The number of chunks to send at once before waiting for confirmation.
+/// More chunks means less latency, fewer chunks means better multiplexing of
+/// RPC calls.
+RAY_CONFIG(uint64_t, object_manager_default_batch_size, 100)
```

Comment:
Looks like a good start, would be interesting to verify if this does indeed fix the broadcast issue. I think we'd probably want a sliding-window (not just batches) for more efficiency in a long term solution though.

Reply:
There's also the question of how to tune this parameter, maybe we just run some experiments and see what a reasonable upper bound is (probably it's not more than a few GB is my guess).

Reply:
Hmm this limit also only applies per object and sender-receiver pair, right? So we could still have a lot more chunks in flight for a given sender depending on what requests it gets. I think we should clarify this in the description.